docker/cloudbuild.yaml

```diff
@@ -58,8 +59,28 @@ steps:
   - '.'
   wait_for: ['20.10.14']
 
+- name: 'gcr.io/cloud-builders/docker'
+  args:
+    - 'build'
+    - '--tag=gcr.io/$PROJECT_ID/docker:24.0.6'
+    - '--file=Dockerfile-versioned'
+    - '--build-arg=${_DOCKER_24}'
+    - '.'
+  id: '24.0.6'
+  wait_for: [ 'base' ]
+
+
+
 # Tests for future supported versions of docker builder go here.
 
+- name: 'gcr.io/$PROJECT_ID/docker:24.0.6'
```

Comment:
@leeonlee did you want this pulled up above this header as well?

Reply:
It would be more consistent to move the step that builds and tags 24.0.6 (lines 62-70) under the "Build all supported versions." header (~line 42)

Reply:
Sorry - got all skewiff. That should all be good now.